bin/roundtrip-translation.sh

```diff
@@ -0,0 +1,393 @@
+#!/bin/bash
```

Comment:
As a general remark, this is one of the clearest written shell scripts I've ever read.  Nice!